.env

```
@@ -0,0 +1 @@
+ REACT_APP_API_KEY=AIzaSyCbnwRIBvWCoQGaLIOFUx4rhLHg-Sb1a8s
```

Now that you're using dotenv, this `.env` file is not intended to be uploaded into your repo. Otherwise the API key is still public and we want to avoid it.

You can, for example, have a `.env.example` file with the variables but not values like:
```suggestion
 REACT_APP_API_KEY=
```

Then keep `.env` file locally for you only.  You can have it listed at the `.gitignore` for not allowing anyone in the team to push it into the repository accidentally.